utils.js

```diff
@@ -334,6 +344,15 @@ function _formatAttachment(attachment1, attachment2) {
         height: attachment1.metadata.dimensions.height,
         duration: attachment1.metadata.duration,
       };
+    case "error":
+      return {
+        type: "error",
+
+        // Save error attachments because we're unsure of their format,
+        // and whether there are cases they contain something useful for debugging.
```

Comment:
Do errors really not have a specific format? Did you encounter those in the wild? If you would you mind sharing the shape of the object.